server/odc-service/src/main/java/com/oceanbase/odc/service/dlm/model/DataDeleteParameters.java

```diff
@@ -33,6 +33,14 @@ public class DataDeleteParameters implements TaskParameters {
 
     private String databaseName;
 
+    private Long targetDatabaseId;
+
+    private String targetDatabaseName;
+
+    private String sourceDataSourceName;
```

Comment:
What does this field do?

Reply:
Because data clearing needs to stay synchronized with data archiving, when there are two data sources, it is necessary to display information about each data source.